<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageDialog</class>
 <widget class="QDialog" name="MessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1615</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
     background:rgb(218,220,224);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableMessageWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <family>Siemens Sans</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">

QTableView
 {
    background-color:  rgb(249,247,248);
   alternate-background-color: rgb(230,230,232);
    color: rgb(55, 56, 59 );
border: 1px solid rgb(249,247,248);
}

QHeaderView::section 
{
 color:rgb(255,255,255); 
 background-color:rgb(64, 77, 83);
 font-family: Siemens Sans;
 font-size: 13px; 
 height: 32px; 
 min-width: 56;
 }

QScrollBar:horizontal 
{
    height: 30px;
    background:  rgb(249, 247, 248 );
}

QScrollBar:vertical
{
     width: 30px;
     background:  rgb(249, 247, 248 );
}

</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frameDashBoardMessageDialog">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777213</width>
       <height>16777214</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
     background:rgb(249,247,248);
}

QLabel{
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
}

QDoubleSpinBox{
  color: rgb(39,51,66);
  font-size: 16px;
  font-weight: bold;
  font-family: Siemens Sans;
}

QDoubleSpinBox::focus{
 border: 2px solid rgb(0,95,135);
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>656</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonClearAllMessages">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
   background-color: rgb(202,51, 51 );
   color: rgb(249, 247, 248 );
   min-height: 40px;
   font-size: 16px;
   font-weight: bold;
   font-family: Siemens Sans;
   border: none;

}
QPushButton:pressed {
     border: 1px solid rgb(0,95,135);    
 }


</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonClose">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
   background-color: rgb(189,192, 197 );
   color: rgb(39, 51, 66 );
   min-height: 40px;
   font-size: 16px;
   font-weight: bold;
   font-family: Siemens Sans;
   border: none;
}

QPushButton:pressed {
     border: 1px solid rgb(0,95,135);    
 
}
QPushButton:disabled {
    background-color: rgb(218,220,224 );
   color: rgb(200,202,206 );
}


</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButtonResetError">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
   background-color: rgb(202,51, 51 );
   color: rgb(249, 247, 248 );
   min-height: 40px;
   font-size: 16px;
   font-weight: bold;
   font-family: Siemens Sans;
   border: none;

}
QPushButton:pressed {
     border: 1px solid rgb(0,95,135);    
 }


</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>655</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
